tools/packaging/packaging.psm1

```diff
@@ -3312,7 +3312,6 @@ function New-GlobalToolNupkg
                 Copy-Item "$LinuxBinPath/*" -Destination $ridFolder -Recurse
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://www.computerperformance.co.uk/powershell/index.htm should work
```powershell
retry of URL failed with error: The SSL connection could not be established, see inner exception.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```

Reply:
@PoshChan please retry static